common4j/src/main/com/microsoft/identity/common/java/providers/microsoft/microsoftsts/MicrosoftStsAuthorizationRequest.java

```diff
@@ -125,6 +125,13 @@ public class MicrosoftStsAuthorizationRequest extends MicrosoftAuthorizationRequ
     // TODO private transient InstanceDiscoveryMetadata mInstanceDiscoveryMetadata;
     // TODO private boolean mIsExtendedLifetimeEnabled = false;
 
+    // The value 1 indicates that client supports DUNA flow.
+    @Getter
+    @Accessors(prefix = "m")
+    @SerializedName("switch_browser")
+    private final String mSwitchBrowser;
```

Comment:
This is meant to be a query parameter, right? How/where is this converted to qp?

Reply:
yes, this qp is sent in the auth request.
the default value is null meaning that when converted to string will not be added.
Only when duna is enabled we set this to 1
https://github.com/AzureAD/ad-accounts-for-android/blob/4a0b3736b8086b86c4e19a45fb13d55efcbc14e8/broker4j/src/main/com/microsoft/identity/broker4j/broker/prt/AbstractPrtAuthorizationStrategy.java#L100